examples/rnn_baselines/train.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2017-present, Facebook, Inc.
```

Comment:
i'm confused it is called rnn_baselines  (plural) but then you just have train.py and agents.py, but i guess they could be moved if we add more rnn baselines?  or could just give them more specific names now